staging/src/kubevirt.io/api/usb/v1alpha1/types.go

```diff
@@ -0,0 +1,50 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// NodeConfig represents a subset of Node resources that we want to expose
+// to virtual machines.
+//
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +genclient
+// +genclient:nonNamespaced
+type NodeConfig struct {
```

Comment:
Is the idea to gradually move all the node information from KubeVirt CR here or will be only for USB devices? I'm asking only because of the name

Reply:
I'm not knowledgeable wrt KubeVirt CR and what moving its node information to NodeConfig entails. What do you think @xpivarc ?

Reply:
If this will stay only for USB devices, I'd suggest renaming it to something more specific like `USBDevicesConfig`. `NodeConfig` might be very confusing

Reply:
Bear in mind this is still under usb subgroup. I don't have anything against generalizing the device plugin's configuration. @vladikr WDYT?